### PR TITLE
Fixes #25400 - remove CV force metadata gen option

### PR DESCRIPTION
**Pull request by jlsherrill**
This removes the abiliy to ignore the optimization
for generation of yum metadata during cv publish
or promote.

### CODE REVIEW

**Comment by jlsherrill** [comment]
See redmine issue for more details

File: app/lib/actions/katello/content_view/promote.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ module ContentView
       class Promote < Actions::EntryAction
         middleware.use Actions::Middleware::KeepCurrentUser
 
-        def plan(version, environments, is_force = false, description = nil, options = {})
+        def plan(version, environments, is_force = false, description = nil, _options = {})
```
Comment (beav) [inline app/lib/actions/katello/content_view/promote.rb]: is `_options` the more correct way to do it?

Comment (jlsherrill) [reply app/lib/actions/katello/content_view/promote.rb]: that's rubocop's suggestion for unused variables.  We could remove that param here, but it seemed safer to just leave it.  I'm open to removing it out right

Comment (beav) [reply app/lib/actions/katello/content_view/promote.rb]: If it's not a lot of investigation to remove it, I'd say just remove it. Better to do it now than later in the 3.10 cycle

**Pull request merged**